python_modules/dagster/dagster/core/types/materializable.py

```diff
@@ -1,17 +1,11 @@
-from dagster import check
-
-
-class Materializeable(object):
-    def define_materialization_config_schema(self):
-        check.failed('must implement')
-
-    def materialize_runtime_value(self, _config_spec, _runtime_value):
-        check.failed('must implement')
+import pickle
 
 
 class FileMarshalable:
```

Comment:
`Pickleable`? also, this plays a little fast and loose with path / file language as written